OpenHAB.Windows/Controls/SetpointWidget.xaml.cs

```diff
@@ -46,6 +47,19 @@ private void ButtonDown_Click(object sender, RoutedEventArgs e)
             Messenger.Default.Send(new TriggerCommandMessage(Widget.Item, value.ToString(CultureInfo.InvariantCulture)));
         }
 
+        private double getDoubleValue(string state)
+        {
+            string newstate = Regex.Replace(state, "[^0-9,.]", string.Empty);
+            double value = 0;
+            try
+            {
+                value = double.Parse(newstate);
```

Comment:
Use double.TryParse(newState) this returns a boolean which indicates if the value was parsable.